mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/query/QueryConverterTest.java

```diff
@@ -50,14 +50,14 @@ public void allCriterionShouldBeWellConverted()throws Exception{
         assertThatJson(queryConverter.from(searchQuery, MAILBOX_UUID).toXContent(jsonBuilder(), QueryBuilder.EMPTY_PARAMS).string())
```

Comment:
Close to 0 :
Have a place in the code where we actually see the query we are issuing, I guess.
In the next version of the code, I deleted them.